subgraphs/compound-v2/src/mappings/helpers.ts

```diff
@@ -375,7 +375,9 @@ export function accrueInterestsOnBalances(market: Market, blockDifference: BigIn
     .times(market.depositRate.div(blocksPerYear))
     .times(blockDifference.toBigDecimal());
   let newInterestBigInt = BigInt.fromString(accruedSupplyInterest.truncate(0).toString());
-  market.inputTokenBalances[0] = market.inputTokenBalances[0].plus(newInterestBigInt);
+  let inputTokenBalance = new Array<BigInt>();
+  inputTokenBalance.push(market.inputTokenBalances[0].plus(newInterestBigInt));
+  market.inputTokenBalances = inputTokenBalance;
```

Comment:
one-liner that applies to all entity array

```
market.inputTokenBalances = [market.inputTokenBalances[0].plus(newInterestBigInt)]
```

Reply:
does this (https://thegraph.com/docs/en/developer/assemblyscript-api/#updating-existing-entities) not apply here?

I've also been doing the longhand way b/c of the comments in ambitious subgraphs pt 2: https://www.youtube.com/watch?v=1-8AW-lVfrA&t=3174s